probe_scraper/probe_expiry_alert.py

```diff
@@ -129,28 +131,14 @@ def parse_args():
 
 
 def main(current_date, dryrun):
+    probe_info = requests.get(PROBE_INFO_BASE_URL + "firefox/all/main/all_probes").json()
```

Comment:
For the schema pipeline, we realized that there was a delay from cloudfront of when the probe-info-service actually updated. To accommodate that we added a parameter to the request with a timestamp, which now forces cloudfront to return the current copy. We should probably do this above as well (for /revisions)
```suggestion
    probe_info = requests.get(f"{PROBE_INFO_BASE_URL}firefox/all/main/all_probes?p={datetime.datetime.now()}").json()
```